server/etcdserver/corrupt_test.go

```diff
@@ -17,6 +17,11 @@ package etcdserver
 import (
 	"context"
 	"fmt"
+	"go.uber.org/zap"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
```

Comment:
Please add a test case with invalid cluster id to `TestInitialCheck`, `TestPeriodicCheck` and `TestCompactHashCheck`.